@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -837,6 +846,45 @@ export class VisibleTileSet {
         tile.dispose();
     }
 
+    /**
+     * Skips rendering of tiles that are overlapped. The overlapping [[Tile]] comes from a
+     * [[DataSource]] which is fully covering, i.e. there it is fully opaque.
+     **/
+    private skipOverlappedTiles(dataSource: DataSource, tile: Tile) {
+        if (this.options.projection.type === ProjectionType.Spherical) {
+            // HARP-7899, currently the globe has no background planes in the tiles (it relies on
+            // the BackgroundDataSource), because the LOD mismatches, hence disabling for globe.
+            return;
+        }
+        if (dataSource.isFullyCovering()) {
+            const ts = dataSource.getTilingScheme();
```

Comment:
Isn't the backround-ds always using webmercator? So the map of maps is not needed